src/Analysis/Solution/Bukkit/Plugin/AuthMeShutdownSolution.php

```diff
@@ -12,10 +13,15 @@
  */
 class AuthMeShutdownSolution extends FileEditSolution
 {
-    protected ?string $path = 'plugins/AuthMe/config.yml';
-    protected bool $relativePath = true;
-    protected ?string $pattern = '/^(\s+)stopServer\: true$/';
-    protected ?string $replacement = '$1stopServer: false';
+    public function __construct(
+        protected string       $path = 'plugins/AuthMe/config.yml',
+        protected FilePathType $type = FilePathType::RELATIVE,
+        protected string       $pattern = '/^(\s+)stopServer\: true$/',
+        protected string       $replacement = '$1stopServer: false'
```

Comment:
These are all properties of the parent, I don't think they should be properties in the childs constructor